src/views/trendingMedia.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+import { getTvShows } from "../utils/http.utils";
+import ToggleButtons from "../components/toggleButtons";
+import ResponsiveAppBar from '../components/responsiveAppBar';
+import CircularProgress from '@mui/material/CircularProgress';
+
+export default function TrendingMedia() {
+    const [tvShows, setTvShows] = useState(null);
+    const [timeWindow, setTimeWindow] = useState('day');
+    const [loading, setLoading] = useState(true)
+
+    useEffect(
+        () => {
+            getTvShows(timeWindow)
+                .then((data) => { setTvShows(data.results); setLoading(false); })
+        }, [timeWindow])
```

Comment:
Since we will make the request again every time the time window changes we need to set the loading to true before making the request again:
```suggestion
    const [loading, setLoading] = useState(false)

    useEffect(
        () => {
            setLoading(true)
            getTvShows(timeWindow)
                .then((data) => {
                    setTvShows(data.results);
                    setLoading(false);
                })
        }, [timeWindow])
```